src/routes/pages/CartPage/CartPage.tsx

```diff
@@ -6,63 +6,53 @@ import ContainerLayout from '../../../components/common/ContainerLayout/Containe
 import CartListTitle from '../../../components/CartListTitle/CartListTitle';
 import CartItem from '../../../components/CartItem/CartItem';
 import CartPriceInfo from '../../../components/CartPriceInfo/CartPriceInfo';
-import OrderButton from '../../../components/OrderButton/OrderButton';
 import EmptyCart from '../../../components/EmptyCart/EmptyCart';
-import Toast from '../../../components/common/Toast/Toast';
-
 import { Logo } from '../../../assets';
 
-import useCartList from '../../../hooks/useCartList';
-import useSelect from '../../../hooks/useSelect';
 import LoadingSpinner from '../../../components/common/LoadingSpinner/LoadingSpinning';
-import { cartPrice } from '../../../utils/cartPrice';
-import { useToastContext } from '../../../context/ToastContext';
 import { useNavigate } from 'react-router';
 import { CartListStyle } from '../../../components/CartList/CartList.styles';
 import CartListHeader from '../../../components/CartList/CartList';
+import { TEXT } from '../../../constants/text';
+import { useCartContext } from '../../../context/CartContext';
+import Button from '../../../components/common/Button/Button';
+import Text from '../../../components/common/Text/Text';
 
 function CartPage() {
-  const cartList = useCartList();
-  const selectedList = useSelect(cartList.data);
-  const { isVisible } = useToastContext();
+  const cart = useCartContext();
   const navigate = useNavigate();
 
-  const totalPrice = cartPrice.totalPrice(
-    cartList.data,
-    selectedList.selectedItems
-  );
-
-  const selectedCartData = cartList.data.filter((item) =>
-    selectedList.selectedItems.includes(item.id)
+  const selectedCartData = cart.data.filter((item) =>
+    cart.selectedItems.includes(item.id)
   );
```

Comment:
하단에서 disabled 여부를 판단하기 위해 쓰이고 있는데요,
이럴 때는 filter 보다 some 혹은 every 를 통해 검사해보세요!
어떤 차이가 있는걸까요?


```suggestion
  const selectedCartData = cart.data.some((item) => cart.selectedItems.includes(item.id));
```